convertHarCsv.js

```diff
@@ -0,0 +1,37 @@
+import * as fs from "fs";
```

Comment:
```suggestion
import fs from "fs";
```
Avoiding wildcard import